application/src/main/java/org/thingsboard/server/actors/ruleChain/RuleNodeActorMessageProcessor.java

```diff
@@ -157,7 +162,7 @@ private TbNode initComponent(RuleNode ruleNode) throws Exception {
     }
 
     @Override
-    protected RuleNodeException getInactiveException() {
-        return new RuleNodeException("Rule Node is not active! Failed to initialize.", ruleChainName, ruleNode);
+    protected RuleNodeException getInactiveException(String data, TbMsgMetaData metaData) {
```

Comment:
Do we really need data and metadata in this case?
If rule node or rule chain is not active, then real root cause happened previously.

Reply:
But probably we can leave it for debug purposes.